core/src/main/java/hudson/slaves/JNLPLauncher.java

```diff
@@ -143,6 +147,21 @@ public boolean isLaunchSupported() {
         return false;
     }
 
+    /**
+     * @since TODO
+     */
+    public boolean isWebSocket() {
```

Comment:
should this be a boolean.  with kafka for remoting and others I am wondering if it should be an enum or something else.

Reply:
The Kafka implementation as I understand it is quite different and does not integrate with `JNLPLauncher`.

See also [discussion](https://github.com/jenkinsci/jep/blob/master/jep/222/README.adoc#dedicated-computerlauncher).